composer.json

```diff
@@ -18,7 +18,7 @@
     "require-dev": {
         "symfony/config": "2.*",
         "phpunit/phpunit": "3.7.*",
```

Comment:
@xabbuh I don't know what you mean, are you talking about `symfony/config`? Isn't this one needed to load the DI config files for the functional tests?